### DISCUSSION TITLE
ideas for new initial config parameters for colouring-core

### DISCUSSION THREAD

**Discussion by traveller195** [tomalrussell/colouring-london#1101] [Ideas]
Hello,
I hope there is not already a similar discussion existing. 
Here, we could collect ideas for further parameters, which could be configured in the beginning of setting up a new Colouring Cities platform from colouring-core.
@matkoniecz @mdsimpson42 @tomalrussell @polly64 

I refer to this: 
https://github.com/colouring-cities/colouring-core/blob/master/docs/configuring-colouring-cities.md

and the following config file:

```
app/src/cc-config.ts
app/src/cc-config.json
```

### First idea for new initial parameter (to find it easier in the beginning):
**extent/ bbox for tile renderer:** 
see 
https://github.com/colouring-cities/colouring-dresden/commit/9b73c3bb7c766147b27275b1594f85bf71d42665

https://github.com/colouring-cities/colouring-core/blob/master/app/src/tiles/rendererDefinition.ts#L26

**Comment by matkoniecz** [comment]
It may make sense to simply create issues for each parameter. "extent/ bbox for tile renderer:" is definitely a good fit for a parameter.

**Comment by traveller195** [comment]
further idea for parameter: (and yes, I/ we can also create issues for that)

URL for city base map layer/ service:

https://github.com/colouring-cities/colouring-core/blob/master/app/src/frontend/map/layers/city-base-map-layer.tsx#L26

as default (for city not in the UK), OpenStreetMap tile service could fit better to start with:
```
const baseUrl = `http://a.tile.openstreetmap.org/{z}/{x}/{y}.png`;
```
problem for the usage as an initial parameter could be, that partly there is the need of an API KEY for OS service, which shoudl be then a second parameter? or maybe just provide a comment where to change the base service, without including it to the config file?
